Floofbot/Modules/Fun.cs

```diff
@@ -168,5 +169,40 @@ public async Task RequestCatFact()
             }
             await Context.Channel.SendMessageAsync(fact);
         }
+
+        [Command("minesweeper")]
+        [Summary("Minesweeper minigame")]
+        public async Task Minesweeper([Summary("width")]int width, [Summary("height")]int height, [Summary("bomb count")]int bombs)
+        {
+           
+            //no negative numbers!
+            if(width < 1 || height < 1 || bombs < 0)
+            {
+                await Context.Channel.SendMessageAsync("Invalid grid size or bomb count");
+                return;
+            }
+            
+            //limits the size of the board
+            if(width > 10 || height > 10)
+            {
+                await Context.Channel.SendMessageAsync("Max Grid Size: 10 x 10");
+                return;
+            }
+            else if(bombs >= height * width)
+            {
+                await Context.Channel.SendMessageAsync("Too many bombs!");
+                return;
+            }
+
+            GameBoard game = new GameBoard(height, width, bombs);
```

Comment:
Sanity check: width and height get switched, here? 
Because the command takes in the width first, followed by the height.